userspace/libsinsp/parsers.cpp

```diff
@@ -1448,6 +1457,27 @@ void sinsp_parser::parse_clone_exit(sinsp_evt *evt)
 	return;
 }
 
+void sinsp_parser::parse_execve_enter(sinsp_evt *evt)
+{
+	sinsp_evt_param *parinfo;
+	char fullpath[SCAP_MAX_PATH_SIZE];
+	uint16_t etype = evt->get_type();
+
+	switch(etype)
+	{
+	case PPME_SYSCALL_EXECVE_18_E:
+		// Get the filename
+		parinfo = evt->get_param(0);
+		sinsp_utils::concatenate_paths(fullpath, SCAP_MAX_PATH_SIZE,
+									   evt->m_tinfo->m_cwd.c_str(), (uint32_t)evt->m_tinfo->m_cwd.size(),
+									   parinfo->m_val, (uint32_t)parinfo->m_len);
+		evt->m_tinfo->m_exepath = fullpath;
```

Comment:
I'm using `concatenate_paths` when parsing the execve filename.
I have noticed that when the parameter is PT_FSPATH it will be parsed using this function on `get_param_as_str` so should I remove this code from here?